src/api/AppApi.js

```diff
@@ -223,16 +220,16 @@ export function getAppByName(appName :String) :Promise<*> {
   * @static
   * @memberof lattice.AppApi
   * @param {UUID[]} appTypeIds
-  * @return {Promise<Schema>} - a Promise that will resolve with a mapping from app type ids to app types
+  * @return {Promise<Object>} - a Promise that will resolve with a mapping from app type ids to app types
   *
   * @example
-  * AppApi.getAppTypeIds(
+  * AppApi.getAppTypeIds([
   *   "ec6865e6-e60e-424b-a071-6a9c1603d735",
   *   "0c8be4b7-0bd5-4dd1-a623-da78871c9d0e"
-  * );
+  * ]);
   */
 
-export function getAppTypeIds(appTypeIds :UUID) :Promise<*> {
+export function getAppTypeIds(appTypeIds :UUID[]) :Promise<*> {
```

Comment:
Perhaps we can rename this function. `getAppTypeIds` is confusing because the function expects `appTypeIds` as a parameter... sooo... what are we actually getting?

Reply:
that is a very good question